Cargo.toml

```diff
@@ -288,7 +288,7 @@ tonic       = { version = "0.13", default-features = false, features = ["prost",
 tonic-build = { version = "0.13", default-features = false }
 tracing     = { version = "0.1" }
 # Local packages used as dependencies
-auth                          = { version = "0.20", path = "src/auth", package = "google-cloud-auth" }
+auth                          = { version = "0.21", path = "src/auth", package = "google-cloud-auth" }
```

Comment:
FYI: we only need to bump the auth version once per release. Bumping it multiple times is harmless though.

(The version bump process is very manual. We just keep track of things in our heads. We would like to refine the process, but it hasn't been a priority yet.)

Reply:
Also, thanks for using `!` in the commit title